cartoframes/auth/credentials.py

```diff
@@ -141,143 +182,15 @@ def delete(self, config_file=None):
             To see if there is a default user credential file stored, do the
             following::
 
-                >>> creds = Credentials()
-                >>> print(creds)
-                Credentials(username=eschbacher, key=abcdefg,
+                >>> credentials = Credentials.from_file()
```

Comment:
Missing `path/to/file.json` in `from_file()`

Reply:
No, it doesn't. We are getting the default file